msteams-platform/bots/how-to/conversations/create-a-bot-commands-menu.md

```diff
@@ -1,12 +1,167 @@
+
+    ![App Studio](Media/AppStudio.png)
+
+2. Once in App Studio, select the __Manifest editor__ tab.
+
+3. In the left column of the manifest editor view in the _Capabilities_ section, select __Bots__.
+
+4. In the right column of the manifest editor view in the _Commands_ section, select the __Add__ button.
+
+    ![App Studio Command Menu Add button](Media/AppStudio-CommandMenu-Add.png)
+
+5. The __New Command__ screen appears. Enter the __Command text__ that you want to have appear as the menu command, and the __Help text__ that you want to have appear directly under the command text in the menu. This should be a brief explanation of the purpose of the command.
+
+6. Next, select the scope(s) that you want this command menu to appear in, then select the __Save__ button.
+
+    ![App Studio Command Menu Add button](Media/AppStudio-NewCommandMenu.png)
+
+The next time you use your bot in teams you will see ___What can I do?___ appear. If you select it, it will drop down the command window that you created. When you select an item it appears in the text box, and you can select the __Send__ button to send the command to your bot for processing.
+
+### Creating a command menu for your bot by editing __Manifest.json__ 
+
+Another valid approach for creating a commend menu is to create it directly in the manifest file while developing your bot source code. Here are a few things to keep in mind when using this approach:
+
+1. Each menu supports up to 10 commands.
+
+2. You can create a single command menu that will work in all scopes.
+
+3. You can create a different command menu for each scope
+
+#### Manifest excerpt - single menu for both scopes
+
+```json
+{
+  ⋮
+  "bots":[
+    {
+      "botId":"[Microsoft App ID for your bot]",
+      "scopes": [
+        "personal",
+        "team"
+      ],
+      "commandLists":[
+        {
+          "scopes":[
+            "team",
+            "personal"
+          ],
+          "commands":[
+            {
+              "title":"Help",
+              "description":"Displays this help message"
+            },
+            {
+              "title":"Search Flights",
+              "description":"Search flights from Seattle to Phoenix May 2-5 departing after 3pm"
+            },
+            {
+              "title":"Search Hotels",
+              "description":"Search hotels in Portland tonight"
+            },
+            {
+              "title":"Best Time to Fly",
+              "description":"Best time to fly to London for a 5 day trip this summer"
+            }
+          ]
+        }
+      ]
+    }
+  ],
+  ...
+}
+```
+
+#### Manifest excerpt - single menu for both scopes
+
+```json
+{
+  ...
+  "bots":[
+    {
+      "botId":"[Microsoft app ID for your bot]",
+      "scopes": [
+        "groupChat",
+        "team"
+      ],
+      "commandLists":[
```

Comment:
In this case you'd want to define a different command in each scope. If you're defining the same command you wouldn't use two separate command lists.

Reply:
The "description" is different for each scope.

This section (Along with its title) come directly from [Add a bot menu in Microsoft Teams](https://docs.microsoft.com/en-us/microsoftteams/platform/concepts/bots/bots-menus#manifest-excerpt---separate-menu-per-scope).